tests/unit/schema/test.action.js

```diff
@@ -0,0 +1,62 @@
+import cloneDeep from 'lodash.clonedeep';
+
+import { validateAddon } from 'schema/validator';
+
+import { validManifest } from './helpers';
+import { assertHasMatchingError } from '../helpers';
+
+describe('/browser_action', () => {
+  it('should be allowed in manifest_version 2', () => {
+    let manifest = cloneDeep(validManifest);
+    manifest = {
+      ...manifest,
+      manifest_version: 2,
+      browser_action: { default_popup: 'popup.html' },
+    };
+    validateAddon(manifest);
+    expect(validateAddon.errors).toBe(null);
+  });
+
+  it('should not be allowed in manifest_version 3', () => {
+    let manifest = cloneDeep(validManifest);
+    manifest = {
+      ...manifest,
```

Comment:
```suggestion
    const manifest = {
      ...cloneDeep(validManifest),
```

Reply:
(maybe)

also do you need `cloneDeep` here?

Reply:
definitely not needed anymore for the top level properties given that we are already creating a new object now, and as long as we don't change any non-toplevel properties coming from the `validManifest` then the cloneDeep would be redundant and not strictly necessary.